qadence/ml_tools/config.py

```diff
@@ -38,6 +38,10 @@ class TrainConfig:
     """
     checkpoint_best_only: bool = False
     """Write model/optimizer checkpoint only if a metric has improved."""
+    val_every: Union[int, None] = None
+    """Calculate validation metric."""
+    epsilon: float = 1e-5
```

Comment:
lets rename `epsilon` to `val_epsilon` for example

Reply:
Done.